src/commands/friends/FriendCommand.ts

```diff
@@ -13,3 +11,101 @@ friend list [@Joker#3650]
 friend requests [incoming/outgoing]
 
 */
+
+import { Command, CommandMessage, CommandoClient } from 'discord.js-commando'
+import { Message, User } from 'discord.js'
+import { oneLine } from 'common-tags'
+import { User as NightwatchUser } from '@nightwatch/db'
```

Comment:
Rename `NightwatchUser` to `BotUser`.

I want to keep names generic in case I need to rebrand the bot.